src/channel.spec.ts

```diff
@@ -164,4 +165,41 @@ describe('Test the channel bridge from iFrame to admin', () => {
     expect(localeMethodMock).toHaveBeenCalledTimes(2);
     expect(fallbackLocaleMethodMock).toHaveBeenCalledTimes(2);
   });
+
+  it('should reject send with missing privileges', () => {
+    send('_privileges', {}).catch(e => {
+      const expectedError = new MissingPrivilegesError('_privileges', ['create:user', 'read:user', 'update:user', 'delete:user']);
+
+      expect(e.message).toEqual(expectedError.message)
+    })
+  });
+
+  it('should not call handle callback with missing extensions', () => {
+    const callback = jest.fn();
+    handle('_privileges', callback)
+
+    // Simulate a postMessage call from an iFrame
+    window.dispatchEvent(new Event('message'))
+
+    expect(callback).toHaveBeenCalledTimes(0)
+  });
+
+  it('should not call handle callback with missing privileges', () => {
```

Comment:
Can you add a test for a positive case where you handle something with privileges and the sender has the right privileges?

Reply:
Will be positve tested in e2e